rules/windows/malware/mal_azorult_reg.yml

```diff
@@ -17,8 +17,8 @@ detection:
     EventID:
       - 12
       - 13
-    TargetObject:
-      - '*SYSTEM\\*\services\localNETService'
+    TargetObject|startswith:
+      - 'SYSTEM\\*\services\localNETService'
```

Comment:
similar problem here:
could be solved this way:

```
    TargetObject|contains: 'SYSTEM\'
    TargetObject|endswith: '\services\localNETService'
```

Reply:
Fixed :)